recipe/meta.yaml

```
@@ -29,7 +31,7 @@ requirements:
 
 test:
   commands:
-    - luarocks config
+    - luarocks config  # [build_platform == target_platform]
```

You should instead add `test: native_and_emulated` to `conda-forge.yaml` *and rerender*.
```suggestion
    - luarocks config
```

---

`conda-forge.yaml` already contains the suggested instructions and was rendered as such.  I'll try applying your suggestion and see if the build passes.